src/scenes/MerchStorePage/PurchaseBox.tsx

```diff
@@ -207,13 +249,24 @@ export default function PurchaseBox({
           {isReceiptState && (
             <>
               <Spacer height={12} />
-              <StyledPurchaseMoreButton
-                onClick={() => {
-                  setIsReceiptState(false);
-                }}
-              >
-                Purchase More
-              </StyledPurchaseMoreButton>
+              {userOwnedSupply === MAX_NFTS_PER_WALLET ? (
+                isMobile ? (
+                  ''
+                ) : (
+                  <StyledOwnMaxText>
+                    You’ve reached the limit of 3 {label}s per collector, and you will not be able
+                    to buy any more.
+                  </StyledOwnMaxText>
+                )
+              ) : (
+                <StyledPurchaseMoreButton
+                  onClick={() => {
+                    setIsReceiptState(false);
+                  }}
+                >
+                  Purchase More
+                </StyledPurchaseMoreButton>
+              )}
```

Comment:
In the receipt state:

If user owns 3 assets:
* If on mobile, render nothing (because this text is shown on `MobileReceiptBox`
* If on desktop, show 'reached limit...' text

If user does not own 3 assets:
* Show purchase more button